Package.swift

```diff
@@ -27,6 +27,7 @@ let package = Package(
         .Package(url: "https://github.com/IBM-Swift/Kitura.git", majorVersion: 1, minor: 3),
         .Package(url: "https://github.com/IBM-Swift/HeliumLogger.git", majorVersion: 1, minor: 3),
         .Package(url: "https://github.com/IBM-Swift/Kitura-MustacheTemplateEngine.git", majorVersion: 1, minor: 1),
-        .Package(url: "https://github.com/IBM-Swift/Kitura-StencilTemplateEngine.git", majorVersion: 1, minor: 3),
+        .Package(url: "https://github.com/IBM-Swift/Kitura-StencilTemplateEngine.git", majorVersion: 1, minor: 4),
+        .Package(url: "https://github.com/kylef/Stencil", majorVersion: 0, minor: 7)
```

Comment:
@youming-lin I think I understand the issue, the change in your example from 2.3 to 2.4 should not break the APIs and module A should not be affected. Omitting a dependency in module A's `Package.swift` practically says: module A does not care which version of module C is fetched due to the dependency in module B's `Package.swift`, if this dependency does not introduce a breaking change in APIs of module C, i.e. the version of module C does not change to 3.0.0. But this can be expressed by specifying the version of module C as 2.x.x (only the major version). 

So how about "saying what you mean" and specifying the major version only? Nothing will be broken until the major version will be increased.

This would work:
```swift
.Package(url: "https://github.com/kylef/Stencil", majorVersion: 0)
```